### PR TITLE
Add loading in details view on contact switch and initialisation 

### PR DESCRIPTION
**Pull request by skjnldsv**
kazam

### CODE REVIEW

File: js/components/contactDetails/contactDetails_controller.js [hunk 1]
```diff
@@ -25,7 +26,8 @@ angular.module('contactsApp')
 		download : t('contacts', 'Download'),
 		delete : t('contacts', 'Delete'),
 		save : t('contacts', 'Save changes'),
-		addressBook : t('contacts', 'Address book')
+		addressBook : t('contacts', 'Address book'),
+		loading : t('contacts', 'Waiting for the contact list to load...')
```
Comment (jancborchardt) [inline js/components/contactDetails/contactDetails_controller.js]: I’d say »Loading contacts« here, that doesn’t sound like we’re complaining about the slowness of our own app ;D

Comment (skjnldsv) [reply js/components/contactDetails/contactDetails_controller.js]: Ahahha, good point!! 😂

Comment (jancborchardt) [reply js/components/contactDetails/contactDetails_controller.js]: Just fixed it ;)

**Comment by skjnldsv** [comment]
You pushed a second before me! 😉👍

**Pull request merged**